### ISSUE TITLE
[QUESTION] Are all of the encoders fine-tuned?

### ISSUE THREAD

**Issue by kayuksel** [Unbabel/COMET#71]
Hello!

Are all of the encoders themselves fine-tuned? If so, I would like to use their embeddings for some experiments.

Sincerely,
Kamer

**Comment by ricardorei** [comment]
Hi @kayuksel! In almost all models the entire XLM-R model is fine-tuned except for the embedding layer. Also, the learning rate is higher on the top layers. This typically helps generalization because lower layers keep most of their original "representations".

**Comment by kayuksel** [comment]
Thank you very much!
